java-plugin/src/main/resources/fr/greencodeinitiative/l10n/java/rules/java/ECJAVA1.json

```diff
@@ -7,7 +7,8 @@
     "constantCost": "50min"
   },
   "tags": [
-    "eco-conception"
+    "ecocode",
```

Comment:
In issue #55, we talked about renaming "eco-conception" to the english equivalent "eco-design".
Maybe we should use it instead of ecoCode (or specify both?)

Reply:
`ecocode` for rule from one of ecoCode plugin, and I will add `eco-design` in addition.

Reply:
@jycr @utarwyn 
ok for `eco-design` (I've just made modifications in all plugins with issue #55 )
for `ecocode`, why not, quite a good idea ... but should we use : 
- `ecocode` : for all ecocode plugins standard an mobile)
- `ecocode` and `ecocode-mobile` to say from what repository rule comes from
- `ecocode-java`, `ecocode-php`, `ecocode-python`, `ecocode-javascript`, `ecocode-ios`, `ecocode-android` to say from what plugin the rule comes from 
I think it will be discussed to have the same tagging system for all our rules

Reply:
I think only `ecocode` label is enough because you can add `ecocode` label + `java` language for rules regarding `ecocode-java`. It remains more flexible to simply keep `ecocode`.